.zenodo.json

```diff
@@ -190,100 +217,139 @@
             "orcid": "0000-0002-0961-9569",
             "affiliation": "Jadavpur University, Kolkata"
         },
+        {
+            "name": "Juanjo Bazán",
+            "orcid": "0000-0001-7699-3983",
+            "affiliation": "CIEMAT, Astroparticle physics, Madrid, Spain."
+        },
+        {
+            "name": "Sarthak Jain"
+        },
         {
             "name": "John Evans"
         },
         {
-            "name": "Michael Malocha"
+            "name": "Sashank Mishra",
+            "orcid": "0000-0001-8302-1584",
+            "affiliation": "Indian Institute of Information Technology, Allahabad"
         },
         {
-            "name": "Monica G. Bobra",
-            "affiliation": "W.W. Hansen Experimental Physics Laboratory, Stanford University, Stanford, CA 94305, USA",
-            "orcid": "0000-0002-5662-9604"
+            "name": "Michael Malocha"
         },
         {
             "name": "Sourav Ghosh",
             "orcid": "0000-0002-7259-5651",
             "affiliation": "Jadavpur University, Kolkata"
         },
         {
-            "name": "Airmansmith97"
+            "name": "sashmish"
```

Comment:
It seems to me that my work username came here unintentionally. If that is the case, then please remove this.